src/formatter/html.ts

```diff
@@ -257,19 +257,20 @@ export class Html {
 		// This regex matches http/https URLs and also patterns like ui5.sap.com/... with or without protocol
 		return cleanedDetails.replace(
```

Comment:
If I'm reading this right, we escape any URL segments that we find with the regex, but the rest of the messageDetails string is left as-is.

Since the messageDetails [_may_ contain user supplied strings](https://github.com/UI5/linter/blob/9931002939b9d4d02b71c948498539d2a2d1fb35/src/linter/messages.ts#L642), I think we should make sure to escape that part of the string as well.

CC: @matz3